### PR TITLE
changes in listener to handle byte messages

### PR DESCRIPTION
**Pull request by Shilpa-Manjunath**

### CODE REVIEW

File: mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java [hunk 1]
```diff
@@ -99,13 +105,17 @@ public boolean consumeLogic(javax.jms.Message message, String abismiddlewareaddr
...
-			messageData = textMessage.getText();
+			if (message instanceof TextMessage || message instanceof ActiveMQTextMessage) {
+				textType = 1;
+				TextMessage textMessage = (TextMessage) message;
+				messageData = textMessage.getText();
+			} else if (message instanceof ActiveMQBytesMessage) {
+				textType = 2;
+				messageData = new String(((ActiveMQBytesMessage) message).getContent().data);
+			} else
+				return false;
```
Comment (MonobikashDas) [inline mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java:117]: will be better if u put logger before returning false

File: mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java [hunk 2]
```diff
@@ -91,6 +96,7 @@
 	private Connection connection;
 	private Session session;
 	private Destination destination;
+	private Integer textType = 0;
```
Comment (MonobikashDas) [inline mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java:99]: create it as a local variable not a global variable

File: mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java (hunk 1 above)
Comment (Shilpa-Manjunath) [reply mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java:117]: ok

File: mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java (hunk 2 above)
Comment (Shilpa-Manjunath) [reply mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java:99]: Done changes

**Pull request merged**